sig-contributor-experience/charter.md

```diff
@@ -1,58 +1,93 @@
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The Contributor Experience Special Interest Group (SIG) is responsible for creating programs and process (even un-processing in some cases) to encourage the health and velocity of the Kubernetes contributor community. Being concentious to our contributor base is critical to scaling the project, growing the ecosystem, and attributes to the projects success. 
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or issues, we take in the feedback and turn it into our project list. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Establish policies, standards, and procedures:
+  - routine GitHub management tasks, including but not limited to: org membership, org permissions, repo creation/administration.
+  - "Org Owner" GitHub permissions, and grant/limit these privileges accordingly
+  - bot accounts, service accounts, webooks, and third-party integrations for all communication platforms that we support but most importantly, GitHub.
+- Establishing a "GitHub Administration team" that will oversee the execution of GitHub management tasks: inviting new members to the org, creating repos, executing moderation decisions, auditing permissions.
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required
+- Deliver and iterate on Thursday community meeting by delivering a variety of content for a weekly overview into the contributor community.
+- Strategize, build, and execute on scalable mentoring programs for all contributor levels.
+- Create best practices, trainings, policies, and moderation of Kubernetes communication platforms. This includes user guidelines when the contributor is the user of the platform. Moderation is an always on, never off service that we supply with trusted contributors spanning time zones. This includes immediate action when dealing with code of conduct issues in a public space that are defined below:
+    - All mailing lists/googlegroups where we have ownership, including k-dev@. At least one Owner of the mailing list must be a communications subproject OWNER. [link to mailing list section of moderation guidelines]
+    - Zoom via CNCF funding [link to zoom guidelines]
+    - YouTube via the /kubernetescommunity channel [link]
+    - Slack via CNCF funding [link]
+    - discuss.kubernetes.io via CNCF funding [link]
+- Help onboard new contributors and current into the culture, workflow, and CI of our contributor experience through the constant addition, modification and retirement of contributor documentation and programs tailored to onboarding.
+- Perform issue triage and maintain the kubernetes/community repository.  
```

Comment:
Do mean issue triage across the whole project? Or just the kubernetes/community repo?

eg: "Maintaining the kubernetes/community repository, including issue triage"